ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/plugin/ExecutionEnvironmentSection.java

```diff
@@ -260,17 +269,53 @@ private void handleRemove() {
 	}
 
 	private void handleAdd() {
-		ElementListSelectionDialog dialog = new ElementListSelectionDialog(PDEPlugin.getActiveWorkbenchShell(), new EELabelProvider());
+		Object[] selection = showEESelectionDialog(false);
```

Comment:
Shouldn't the argument be `true`?
```suggestion
		Object[] selection = showEESelectionDialog(true);
```